gemsfx/src/main/java/com/dlsc/gemsfx/StripView.java

```diff
@@ -232,6 +232,24 @@ public void scrollTo(T item) {
         getProperties().put("scroll.to", item);
     }
 
+    /**
```

Comment:
The java docs have to be added to the property() method, in this case loopSelectionProperty().